test/functional/export_controller_test.rb

```diff
@@ -1,71 +1,102 @@
 require 'test_helper'
 
 class ExportControllerTest < ActionController::TestCase
-
   def setup
     @map = maps(:saugus)
   end
 
   def teardown
   end
 
-  test "index" do
+  test 'index' do
```

Comment:
Maybe a more significant and standardized test names would be bettter, such as `test 'should visit Export index page'`